src/groovy/org/pih/warehouse/core/Constants.groovy

```diff
@@ -144,4 +144,7 @@ class Constants {
     static final String ROOT_CATEGORY_NAME = "ROOT"
 
     static final String DEFAULT_ORGANIZATION_CODE = "ORG"
+
+    // Order number suffix for bin replenishment case
+    static final String REPLENISHMENT_CODE = "_R"
```

Comment:
Should be REPLENISHMENT_PREFIX = "R-"
Add it also in the config as:
`openboxes.stockTransfer.binReplenishment.prefix = Constants.REPLENISHMENT_PREFIX`